docs/kubefirst/overview.md

```diff
@@ -1,48 +1,39 @@
-# Enterprise offering
+# What is Kubefirst
 
-### Thousand foot view
+Kubefirst is a fully automated and operational open source platform that includes some the best tools available in the 
+Kubernetes space, all working together from a single command. By running `kubefirst cluster create` against your empty 
+aws cloud account, you'll get a GitOps cloud management and application delivery ecosystem complete with automated 
+terraform workflows, vault secrets management, **GitLab** or **GitHub** integrations with Argo, and demo applications 
+that demonstrates how it all pieces together.
 
-The kubefirst team has the ability to pro plan delivers 3 AWS accounts: `mgmt`, `preprod`, and `production`. Everything delivered in these 3 accounts has been sourced from the `gitops` repository which houses all of our Terraform IAC and Argo CD gitops configurations.
+## Install overview
 
-It includes Source Control, Single Sign On, Secrets Management, 
-Automated Infrastructure as Code, Helm Package Management, GitOps Delivery, 
-Managed Kubernetes, Container Registry, Automated SSL Certificates, Container Scanning, 
-User Management, An Example Demo Application, Application Monitoring, Logging & Observability, 
-and a Docs and Blogging platform for your internal or external communications.
+[//]: # (todo: update wording)
+- The `kubefirst` cli runs on your localhost and will create an **AWS EKS cluster** that includes **GitLab** or **GitHub**, **Vault**, **ArgoCD** and **Argo Workflow**, and example applications with the **Metaphors** application to demonstrate how everything on the platform works.
+- The install takes about 30 minutes to execute. Day-2 operations can't commonly be done within the same hour of cluster provisioning. Kubefirst is solving this on our open source platform. We really hope that's worth a [GitHub Kubefirst repository](https://github.com/kubefirst/kubefirst) star to you (top right corner).
+- Your self-hosted GitLab or cloud GitHub will come preconfigured with two git repositories `kubefirst/gitops` and `kubefirst/metaphor-frontend`, `kubefirst/metaphor-go`, `kubefirst/metaphor`.
+- All of the infrastructure as code will be in your GitOps repository in the Terraform directory. IAC workflows are fully automated with atlantis by merely opening a merge request against the `gitops` repository.
+- All of the applications running in your kubernetes cluster are registered in the GitOps repository in the root `/registry` directory.
+- The Metaphors repositories only needs an update to the main branch to deliver the example application to your new development, staging, and production environments. It will hook into your new Vault for secrets, demonstrate automated certs, automated DNS, and GitOps application delivery. Our CI/CD is powered by ArgoCD, Argo Workflows, GitLab or GitHub, and Vault.
```

Comment:
```suggestion
- The Metaphors repositories only needs an update to the main branch to deliver the example application to your new development, staging, and production environments. It will hook into your new Vault for secrets, demonstrate automated certs, automated DNS, and GitOps application delivery. Our CI/CD is powered by Argo CD, Argo Workflows, GitLab or GitHub, and Vault.
```